cmd/tink-server/Dockerfile

```diff
@@ -6,6 +6,5 @@ EXPOSE 42114
 ARG TARGETARCH
 ARG TARGETVARIANT
 
-RUN apk add --update ca-certificates && \
-    apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing cfssl
+RUN apk add --update ca-certificates
```

Comment:
Should we consider using a --no-cache (supported in alpine linux >= 3.3) or --update followed by the command rm -rf /var/cache/apk/* when apk adding packages. This will result in a smaller image size in my experience

Reply:
Last time I checked on this it was not necessary. apk won't cache if its cache directory doesn't exist (which don't in base images). I believe its the same with --update. Let me check though.

Reply:
Ok, let me know

Reply:
ok I've checked `--no-cache` makes a difference (albeit small), seems like the APKINDEX files do get saved without it (I was thinking of the .apk files themselves). FYI:
```
[16:45:10]-[~]─[manny@zennix]> 
echo 'FROM alpine:3.15
RUN apk add --update ca-certificates' | docker build -t mmlb-alpine-testing-update - >/dev/null 2>/dev/null
                                                                                                                                                                      
[16:45:22]-[~]─[manny@zennix]> 
echo 'FROM alpine:3.15
RUN apk add --update --no-cache ca-certificates' | docker build -t mmlb-alpine-testing-update--no-cache - >/dev/null 2>/dev/null
                                                                                                                                                                      
[16:46:01]-[~]─[manny@zennix]> 
echo 'FROM alpine:3.15
RUN apk add --update ca-certificates && rm -rf /var/cache/apk/*' | docker build -t mmlb-alpine-testing-update-plus-rm - >/dev/null 2>/dev/null
                                                                                                                                                                      
[16:46:08]-[~]─[manny@zennix]> 
docker images | grep mmlb-alpine-testing
mmlb-alpine-testing-update-plus-rm                              latest                             d3fc2ab90e39   6 seconds ago    6.06MB
mmlb-alpine-testing-update--no-cache                            latest                             dc7942383116   46 seconds ago   6.06MB
mmlb-alpine-testing-update                                      latest                             1dd3110c6076   57 seconds ago   8.36MB
```

Reply:
I say we add it and try to standardize on it always being there since some of the images can get beefy

Reply:
+ there is no harm in having it, as opposed to the other way around

Reply:
Updated @markyjackson-taulia ptal